README.md

```diff
@@ -257,6 +257,10 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // If provided, it MUST be base64url encoded.
                 "nonstandard_id_token_secret": "xxxxxxxxxxxxxxx",
 
+                // Determines if the "aud" member is required in the
+                // response from the introspection endpoint.
+                "require_aud_member_from_introspection_endpoint": false,
```

Comment:
Let's adjust this description to use wording like the other config options.
```js
// Controls whether the HTTP API requires the presence of the
// "aud" member in the introspection endpoint response.
```

Reply:
Sounds good, I'll add it in a moment.

Reply:
Updated the wording a bit.

Reply:
Looks good. Resolving.